lib/modules/dosomething/dosomething_signup/includes/SignupTransformer.php

```diff
@@ -75,6 +73,8 @@ protected function transform($item) {
 
     $data['campaign'] = $this->transformCampaign((object) $item->campaign);
 
+    $data['reportbacks'] = $this->transformReportback($item->rbid);
```

Comment:
Can we change this to just `reportback` ? Each Signup will only have one Reportback, not multiple.